src/collectors/eventstoreprojections/eventstoreprojections.py

```diff
@@ -0,0 +1,110 @@
+# coding=utf-8
+
+"""
+A Simple collector which calls the Eventstore API for
+the status of projections and parses this to flat metrics.
+String values are ignored, except for the name and status.
+Name is used for the metric path, status is translated to
+an integer (running = 1, stopped = 0).
+Note: "$" are replaced by underscores (_) in the projection
+name to avoid problems with hostedGraphite/Grafana.
+
+This collector is based upon the HTTPJSONCollector.
+
+#### Dependencies
+
+ * urllib2
+
+"""
+
+import urllib2
+import json
+import diamond.collector
+
+
+class EventstoreProjectionsCollector(diamond.collector.Collector):
+
+    def get_default_config_help(self):
+        config_help = super(
+            EventstoreProjectionsCollector, self).get_default_config_help(
+        )
+        config_help.update({
+            'path': "name of the metric in the metricpath",
+            'url': 'URL of the evenststore instance',
+            'route': 'route in eventstore for projections',
+            'port': 'tcp port where eventstore is listening',
+            'headers': 'Header variable if needed',
+            'debug': 'Enable or disable debug mode',
+        })
+        return config_help
+
+    def get_default_config(self):
+        default_config = super(
+            EventstoreProjectionsCollector, self).get_default_config(
+        )
+        default_config.update({
+            'path': "eventstore",
+            'url': 'http://localhost',
```

Comment:
So I still feel like this option should be `hostname` instead of url. Might be best to break it up into 2 options `protocol` and `hostname`